workspaces/ballerina/ballerina-visualizer/src/views/BI/SamplesView/index.tsx

```
@@ -0,0 +1,389 @@
+const CATEGORY_COLOR_MAP: Record<string, string> = {
+    "Automation": "#4A90E2",
+    "File Integration": "#FF8C42",
+    "Integration": "#52C41A",
+    "AI Agent": "#9C27B0",
+    "Integration as API": "#00BCD4",
+    "Default": "#4A90E2"
+};
+
+interface SamplesViewProps {
+    // Samples view props are not needed
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)